media_driver/linux/common/os/mos_context_specific.cpp

```diff
@@ -475,26 +475,6 @@ void OsContextSpecific::SetSliceCount(uint32_t *pSliceCount)
         }
     }
 
-    struct drm_i915_gem_context_param_sseu sseu = { .flags = I915_EXEC_RENDER };
```

Comment:
I suggest to remove all the code under MEDIA_EXT for SSEU. Both here in this function and everywhere across the driver.